plugins/gatsby-plugin-eol-rss/gatsby-node.js

```diff
@@ -0,0 +1,127 @@
+const preferDefault = (m) => (m && m.default) || m;
```

Comment:
just curious about this, does the parseISO require not work without it? why does format not need it?

Reply:
I'm not sure actually, I'm gonna get rid of it. Long term effects of a c/p